subprojects/gradle/impact-shared/src/main/java/com/avito/impact/configuration/ImplementationConfiguration.kt

```diff
@@ -58,6 +59,10 @@ class ImplementationConfiguration(module: InternalModule) : SimpleConfiguration(
                     .dependencies
                     .withType(DefaultProjectDependency::class.java)
             }
+            .filter {
+                val category = it.attributes.getAttribute(Category.CATEGORY_ATTRIBUTE)?.name
+                category == null || category == Category.LIBRARY
```

Comment:
What deps are with category null?

Reply:
most of them is null (i just used debugger, don't know much)